tests/formats/mmtf.cpp

```diff
@@ -189,6 +189,22 @@ TEST_CASE("Read files in MMTF format") {
         CHECK(!topology.are_linked(topology.residue(0), topology.residue(2)));
     }
 
+    SECTION("Large MMTF file") {
+    // Test fails on Windows due to timing of MSVC debug builds
+    #ifndef CHEMFILES_WINDOWS
```

Comment:
It would make sense to have benchmarks separated from tests. This can stay as is now, I'll check how we can have benchmarks when I find the time

Reply:
Looks like Catch (which we are using for tests) supports benchmarks too: https://github.com/catchorg/Catch2/blob/master/docs/benchmarks.md

Reply:
Yeah, that's probably a good idea. We should be benchmarking the release builds, not the debug ones.